back-end/services/nodemailer.js

```diff
@@ -42,9 +43,12 @@ const sendMail = async (emailTo, name) => {
     },
   });
 
-  const mailOptions = registrationMailOptions(name);
+  const options =
+    registrationOrSubscription === REGISTRATION
```

Comment:
Use `switch case` for handling more conditions. You could use `SUBSCRIPTION` constants for this